gcloud/pubsub/client.py

```diff
@@ -50,6 +50,12 @@ def list_topics(self, page_size=None, page_token=None):
         See:
         https://cloud.google.com/pubsub/reference/rest/v1/projects.topics/list
 
+        Example:
+
+        .. literalinclude:: pubsub_snippets.py
+           :start-after: [START client_list_topics]
+           :end-before: [END client_list_topics]
```

Comment:
I said "infeasible", not "impossible":  the trick for testing the snippets against actual API calls is that there is lots of setup / teardown required:  signalling what that should be from *inside* the docstring would damage the user experience.  If we don't test them (frequently/automatically), then we live with bitrot.

At lunch today, @mcdonc (my business partner, Chris McDonough) suggested a mode where the `.. literal-include::` blocks in `__doc__` could be post-processed at application startup (maybe via an environment variable) to pull in the examples.